mpt_extension_sdk/api/builders/event.py

```
@@ -102,12 +106,14 @@ def create_non_task_event_route(  # noqa: WPS213
     handler_logger = logging.getLogger(route.callback.__module__)
 
     @router.post(route.path, status_code=status.HTTP_200_OK, response_model=EventResponse)
-    async def handle_event(event: Event) -> EventResponse:  # noqa: WPS430,WPS213
+    async def handle_event(request: Request, event: Event) -> EventResponse:  # noqa: WPS430,WPS213
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS430, WPS213

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)